.github/workflows/ci.yml

```diff
@@ -217,7 +217,7 @@ env:
 jobs:
   test-and-build:
     name: Test and build plugin
-    runs-on: ubuntu-latest-8-cores
+    runs-on: ubuntu-x64-large
```

Comment:
This is "large" (as well as the runner used for Playwright e2e tests) for backwards compatibility (self-hosted large has 8 cores, vs the default 4). There's an issue to allow customizing the size of the runner (#97)